packages/docs/docs/app/_category_.json

```diff
@@ -1,4 +1,4 @@
 {
   "label": "App",
-  "position": 4
+  "position": 5.1
```

Comment:
I totally understand this, but it also feels a little stinky.  I want to research how other big docusaurus sites manage this.

Reply:
The "real" way to do this is to configure a sidebar manually in the `docusaurus.config.json` file. Would like to do that in a separate re-factoring pass

Reply:
For example, take a look at supabase: https://github.com/supabase/supabase/blob/master/web/sidebars.js